src/main/resources/i18n/MessageBundle.properties

```diff
@@ -0,0 +1,49 @@
+application.name = TimeSheetGenerator
+
+command.input.help.description = Prints helping information
+command.input.version.description = Prints the version
+command.input.gui.description = Enables load/save dialogs
+command.input.file.description = Passes file paths via console
+command.input.file.arguments = global.json> <month.json> <output-file.tex
+command.output.version = Version: {0}
+command.output.buildInfo = Built from {0} ({1}) at {2}
+
+file.tex.extension = tex
+file.tex.description = TeX File
+file.tex.output.description = Output Tex File
+file.json.extension = json
+file.json.description = JSON File
+file.json.global.description = Global JSON File
+file.json.month.description = Month JSON File
+
+locale.timespan.parseRegex = ^[0-9]+:[0-5]?[0-9]$
+locale.timespan.separatorHourMinute = :
+locale.timespan.stringFormat = {0,number,00}:{1,number,00}
+
+error.checker.totalTimeExceedance = Maximum legal working time exceeded.
+error.checker.dayTimeExceedance = Maximum daily working time of {0} exceeded on {1,date}.
+error.checker.timeOutOfBounds = Working time is out of bounds on {0,date}.
+error.checker.timeSunday = The {0,date} is a sunday, which is not a valid working day.
+error.checker.timeHoliday = The {0,date} is an official holiday, which is not a valid working day.
+error.checker.timePause = Maximum working time without pause exceeded on {0,date}.
+error.checker.timeOverlap = Start/End times in the time sheet overlap on {0,date}.
+error.checker.rowNumExceedance = Exceeded the maximum number of rows for the document.
+error.checker.nameMissing = Name of the departement is missing.
+
+error.entry.timeOverUpperLimit = Start and end time may not be greater than 23:59.
+error.entry.startGreaterThanEnd = Start time may not be greater than end time.
+error.timesheet.sumOfTimeNegative = Sum of predTransfer and vacation cannot be greater than sum of maxWorkingTime and succTransfer.
+error.timespan.timeNegative = Hour and minute may not be negative.
+error.timespan.minuteOverUpperBound = Minute may not be greater than {0,number,integer}.
+error.timespan.subtrahendGreaterThanMinuend = Subtrahend may not be greater than Minuend.
+error.timespan.invalidParseInput = Invalid time string. Usage: h...h:mm
+error.workingarea.invalidParseInput = Invalid string: Cannot be parsed to WorkingArea.
+
+error.contextstringreplacer.emptyReplaceString = Strings to replace may not be empty.
+
+error.userinput.mutuallyExclusiveOptionsGuiFile = GUI and file option cannot be used at the same time.
+error.userinput.versionNotFound = Version not found.
+error.userinput.fileCouldNotBeOpened = File could not be opened.
+error.userinput.fileDoesNotExist = Selected file does not exist.
+error.userinput.unsupportedException = Selected file has an unsupported extension.
```

Comment:
Is this named "unsupportedException" because its used in an exception or would it be clearer to name it "unsupportedExtension" :)

Reply:
It should've been "unsupportedExtension", but somehow I didn't type it :D